test/geometry_tests.cpp

```diff
@@ -162,6 +204,48 @@ TEST(Point2DPlanning_Griz_Test, bresenham_conversion_octant1_test) {
 
 }
 
+TEST(Point2DPlanning_Griz_Test, raytraceLine_conversion_pointTest) {
+	Cell startPix = {.x = 0, .y = 0};
+	Cell endPix = {.x = 0, .y = 0};
+	auto pixels = raytraceLine_conversion(startPix, endPix);
+
+	// printPixels(pixels);
+
+	ASSERT_EQ(pixels.size(), 1);	
```

Comment:
Prefer `EXPECT_` unless there would be no point in continuing the test afterwards

Reply:
Done